packages/fxa-auth-server/lib/senders/emails/templates/cadReminderSecond/index.stories.ts

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Story, Meta } from '@storybook/html';
+import storybookEmail, { StorybookEmailArgs } from '../../storybook-email';
+import * as cadReminderFirst from '../cadReminderFirst/index.stories';
+
+export default {
+  title: 'Emails/cadReminderSecond',
+} as Meta;
+
+const Template: Story<StorybookEmailArgs> = (args) => storybookEmail(args);
+
+const defaultVariables = {
+  ...cadReminderFirst.CadReminderDesktopTablet.args.variables,
+  bodyText:
```

Comment:
I think you can take this line out. `bodyText` is identical to what you've got in `cadReminderFirst` vars so we shouldn't need to override it.

Reply:
`cadReminderFirst` has this extra text: `It takes two to sync.` in body so is it okay if we crop it out of `cadReminderSecond`?